src/model/uniprotkb/sections/StructureConverter.ts

```diff
@@ -1,18 +1,41 @@
 import FeatureType from '../../types/FeatureType';
-import { convertSection } from '../SectionConverter';
+import { convertSection, UIModel } from '../SectionConverter';
 import EntrySection from '../../types/EntrySection';
 import { UniProtkbAPIModel } from '../UniProtkbConverter';
+import { groupBy } from '../../../utils/utils';
+import { Xref } from '../../types/CommentTypes';
+
+export type StructureUIModel = {
+  structures?: Map<string, Xref[]>;
+} & UIModel;
 
 const featuresCategories = [FeatureType.HELIX, FeatureType.STRAND];
 
 const convertStructure = (data: UniProtkbAPIModel) => {
-  return convertSection(
+  const structureData: StructureUIModel = convertSection(
     data,
     undefined,
     undefined,
     featuresCategories,
     EntrySection.Structure
   );
+  // Extract xrefs to PDB
+  if (data.databaseCrossReferences) {
+    const structureInfo = data.databaseCrossReferences
+      .filter(ref => ref.databaseType === 'PDB')
+      .map(item => {
+        const method =
+          item.properties &&
+          item.properties.find(property => property.key === 'Method');
+        return { ...item, method: method && method.value };
```

Comment:
`method` could potentially undefined here (if `item.properties` doesn't exist or `item.properties.find` finds nothing`. Will this be problematic?

Reply:
Changed to an empty string - this is used to create the groupings so if it's missing it will be grouped by empty string which respects the type.